support/kmm.spec.yaml

```diff
@@ -92,9 +92,10 @@ spec:
               message: The API deployment does not have any ready replicas.
           - pass:
               message: There is at least one Ready replica of the Deployment.
+    #FIXME: does this one need to be updated manually?
```

Comment:
It does. I would move to https://troubleshoot.sh/docs/analyze/cluster-pod-statuses/ with the appropriate selectors.

Reply:
Done.